Options.py

```diff
@@ -427,7 +428,6 @@ def __init__(self, value: typing.Union[str, int]):
         assert isinstance(value, str) or isinstance(value, int), \
             f"{value} is not a valid option for {self.__class__.__name__}"
         self.value = value
-        super(TextChoice, self).__init__()
```

Comment:
This seems like a random change to be in this PR

Reply:
this turned out to break the subclasses which i missed since i was overriding this in the Bosses class. I addressed this in a different PR and just mimicked it here to save me some time

Reply:
which PR is that?

Reply:
#1044